app/src/main/java/org/fossasia/openevent/general/search/SearchTimeActivity.kt

```diff
@@ -17,6 +17,7 @@ const val TODAY = "Today"
 const val TOMORROW = "Tomorrow"
 const val THIS_WEEKEND = "This Weekend"
 const val NEXT_MONTH = "In the next month"
+private const val SEARCHTIME: String = "time"
```

Comment:
Duplication. Make the previous constant public and remove this from here. The reason we use constants is to remove duplication of writing strings which are error prone